profile.py

```diff
@@ -150,7 +162,15 @@ def right_penumbra(self, upper=0.9, lower=0.1):
     def center(self, level=0.5):
         return 0.5 * (self.x_at_y(level) + self.x_at_y(level, reverse=True))
 
-    def mirror(self):
+    def mirror(self, m=0):
```

Comment:
Add doctest, in similar manner as in x_at_y. 
Each test should be one line of the code, sth like:

```
print(Profile([....]).mirror())
[...]
```